crates/burn-cubecl-fusion/src/matmul/builder.rs

```diff
@@ -67,8 +68,8 @@ impl<R: Runtime> OptimizationBuilder<CubeOptimization<R>> for MatmulBuilder<R> {
                 let out = self.builder.output_unhandled(&op.out);
 
                 self.matmul = Some(FusedMatmul::new(
-                    lhs,
-                    rhs,
+                    MatmulArg::Normal(lhs),
+                    MatmulArg::Normal(rhs),
```

Comment:
I would be cool to support quantized

Reply:
I've added my best guess at an implementation, but can't test at the moment